### PR TITLE
JupyterHub tightened up w/ python3-psutil, --system-site-packages [python3-venv also removed from Calibre-Web & Lokole] [Node.js apt source pollution detox]

### PR DESCRIPTION
**Pull request by jvonau**
### Fixes bug:
#3553 https://github.com/iiab/iiab/pull/3549#issuecomment-1523231127
### Description of changes proposed in this pull request:
same fix as calibre-web
### Smoke-tested on which OS or OS's:
Pending

### CODE REVIEW

File: roles/calibre-web/tasks/install.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@
   package:
     name:
       - imagemagick
-      - python3-venv
```
Comment (holta) [inline roles/calibre-web/tasks/install.yml:5]: Trim Line 1 too?

File: roles/lokole/tasks/install.yml [hunk 2]
```diff
@@ -7,7 +7,6 @@
     name:
       #- python3    # 2022-12-21: IIAB pre-req, see scripts/local_facts.fact
       #- python3-pip
-      - python3-venv
```
Comment (holta) [inline roles/lokole/tasks/install.yml:10]: Change Line 5 from 13 to 12 ?

**Comment by jvonau** [comment]
http://sprunge.us/Zjrgbr?en at eacfec8
sudo ./runrole --reinstall nodejs
sudo ./runrole --reinstall jupyterhub
http://sprunge.us/9waMgZ?en at 0f8fdb7

**Comment by jvonau** [comment]
It's not a bug just a workaround for a wheel that is not available from pypi for python 3.11

**Comment by jvonau** [comment]
works see https://github.com/iiab/iiab/pull/3549#issuecomment-1523461082 http://sprunge.us/Ve5UBz?en
```
 162 [jupyterhub]
 163 name = JupyterHub
 164 description = "High Schools may want to consider JupyterHub to integrate coding with dynamic interactive graphing — A New Way to Think About Programming — allowing students to integrate science experiment results and program output within their notebook/document/blog."
 165 jupyterhub_install = True
 166 jupyterhub_enabled = True
 672 jupyterhub_installed: True
```

**Comment by holta** [comment]
Well-tested and looks great!

Thanks to @EMG70 for the careful testing on several different platforms:

- Ubuntu Desktop 23.04 on RPi 400
- RasPiOS Desktop on RPi 400
- Debian 11 on x86_64 laptop

**Comment by holta** [comment]
Tangentially related:

- PR iiab/iiab-admin-console#539

**Pull request merged**